binding/angular/projects/rhino-web-angular/src/lib/rhino.service.ts

```diff
@@ -108,6 +115,7 @@ export class RhinoService implements OnDestroy {
     const {
       accessKey,
       context,
+      endpointDurationSec,
       requireEndpoint,
```

Comment:
can we make these default to '1.0' and 'true'?

Reply:
the same here :)